msprime/mutations.py

```diff
@@ -795,7 +806,10 @@ def mutate(
     :param bool discrete: Whether to generate mutations at only integer positions
         along the genome.  Default is False, which produces infinite-sites
         mutations at floating-point positions.
-    :return: The :class:`tskit.TreeSequence` object  resulting from overlaying
+    :param bool allow_ancestral: Whether to allow mutations to be added
+        ancestrally to existing (kept) mutations. This flag has no effect
```

Comment:
modify this to "whether to allow start time before existing mutations" or somethign

Reply:
change to "allow_kept_mutations_after_start_time" or "kept_after_start_time_ok" "kept_after_start_time"...?